<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Reboot</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Shutdown</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Meminfo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>MemUsed:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>KB</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Text</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_12">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>SwapTotal:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QProgressBar" name="progressBar">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="value">
           <number>24</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>KB</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>SwapUsed:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="5">
         <widget class="QChartView" name="chartview"/>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>MemTotal:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Swap</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>KB</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>MemAvailable:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>KB</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>KB</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QProgressBar" name="progressBar_2">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Memory</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Cpuinfo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>CPU0:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QProgressBar" name="progressBar_3">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>CPU1:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QProgressBar" name="progressBar_4">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QChartView" name="chartview_2"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>CPU0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>CPU1</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Processinfo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>Pid or Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Details</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTableWidget" name="tableWidget">
          <property name="rowCount">
           <number>50</number>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Kill</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>80</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>130</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>180</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>238</y>
          <width>351</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>22</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
